attributes/default.rb

```diff
@@ -52,7 +52,7 @@
 default['openssh']['server']['gssapi_authentication'] = 'yes'
 default['openssh']['server']['gssapi_clean_up_credentials'] = 'yes'
 default['openssh']['server']['x11_forwarding'] = 'yes'
-default['openssh']['server']['subsystem'] = 'sftp /usr/libexec/sftp-server'
+default['openssh']['server']['subsystem'] = 'sftp /usr/libexec/openssh/sftp-server'
```

Comment:
Can you confirm that path is correct for all the supported AMIs?

Reply:
Ya just confirmed. Seems path is different on ubuntu (/usr/lib/openssh/sftp-server | /usr/lib/sftp-server) and for centos / alinux (/usr/libexec/openssh/sftp-server) 

Will update the PR

Reply:
Seems debian has override already https://github.com/awslabs/cfncluster-cookbook/commit/1a8856ce388df19e4e2847f76511762b343bfcdf

This PR looks good to me now.